android/src/com/mapswithme/maps/widget/placepage/PlacePageButtons.java

```diff
@@ -308,22 +310,24 @@ private void preserveRoutingButtons(@NonNull List<PlacePageButton> items, @NonNu
     }
   }
 
-  private ArrayList<MenuBottomSheetItem> getMenuItems(final List<PlacePageButton> buttons)
+  public ArrayList<MenuBottomSheetItem> getMenuBottomSheetItems()
   {
+    if (mMoreItems == null)
+      return null;
     ArrayList<MenuBottomSheetItem> items = new ArrayList<>();
-    for (int i = mMaxButtons; i < buttons.size(); i++)
+    for (int i = mMaxButtons; i < mMoreItems.size(); i++)
     {
-      PlacePageButton bsItem = buttons.get(i);
+      PlacePageButton bsItem = mMoreItems.get(i);
```

Comment:
Please try to use the `final` keyword everywhere if the variable is not modified. It helps to read the code and understand if the variable is "const".